### DISCUSSION TITLE
Remove Scrollbar Area Reserved

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#708] [Ideas]
![scrollbararea](https://user-images.githubusercontent.com/6903107/135071921-a7c4d394-f516-41ea-8101-333b7d037d5d.png)
Does the autohidescrollbar setting work for this? The scroll bar area is reserved and does not disappear, but when a scroll bar is not required (when the number of list items is small), it would be better to disappear for design. It was a difficult part to fix with my skill.

**Comment by jjw24** [comment]
I am ok with it either way as long as they are symmetrical but am leaning towards removing the spacing on the right though so it looks better and scrollbar can flow on top and/or autohidden.

**Reply by onesounds** [reply]
When I first edit this part, Contrary to what I thought, there were some problems.
1. If I Remove scrollbar area, middle area shaking when I type query. because middle is responsible width. (feeling was not good) and I think using fixed width is not good choice.
2. when I choose the last item with scrollbar. there are still has space. 

So I thought a lot, but I changed it to the following.

- I changed Scrollbar positioning to middle column and aligment to right.
- Different Thumb/Selected area color
- Change Margin by computescrollbarvisibility.

Anyway, Now It works. 
I think it's beautiful(Not code...) and moves well.
